README.md

```diff
@@ -648,6 +648,15 @@ Another package may select this version by specifying:
 
 Also, `/// <reference types=".." />` will not work with path mapping, so dependencies must use `import`.
 
+#### How do breaking type changes work if type declaration packages closely track the library package's version?
```

Comment:
If this is for consumers, I think this title is more meaningful:

```suggestion
#### If type declaration packages track the library package's major and minor version, why did a non-major version bump break?
```

but I think Jake's question about the problem solved by this PR needs to be addressed before it's worth tweaking the wording.

Reply:
As of [this comment](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/68177#discussion_r1450975516), we started on a direction of "written for contributors, but readable by consumers". I pushed the update, which I think still works well with existing proposed title. What do you think?